### ISSUE TITLE
Warn on empty term, do not process those

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod-utils#75]
From https://github.com/metanorma/iso-10303-2/issues/30 there is an empty term entered:

```adoc
// STEPmod path: data/modules/mechanical_design/module.xml/
// CVS working rev: (1.8), repo rev (1.8)
// CVS: revision up to date
===

[.source]
<<ISO-TS_10303-1846_2017,clause=3.1>>
```

stepmod-extract-terms should issue a warning to the user and ignore empty terms.

_Originally posted by @manuel489 in https://github.com/metanorma/iso-10303-2/issues/27#issuecomment-832312435_

**Comment by TRThurman** [comment]
I agree that is a sensible solution.

**Comment by ronaldtse** [comment]
Thanks @TRThurman !

**Comment by ronaldtse** [comment]
This is fixed in 396d46c3fcef8526a50ddc03e36f286f141f3ffa.
